.devcontainer/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.195.0/containers/php/.devcontainer/base.Dockerfile
```

Comment:
It seems to me like doing a devcontainer here, while admirable, is out of scope. Could this be done in a separate PR?